Mathlib/Topology/Order/ScottTopology.lean

```diff
@@ -259,7 +259,7 @@ lemma continuous_monotone {f : WithScottTopology α → WithScottTopology β}
 
 end WithScottTopology
 
-lemma ScottContinuous_iff_continuous_wrt_Scott
+lemma ScottContinuous_iff_continuousWrtScott
```

Comment:
`ContinuousWrtScott` isn't a declaration, so this name looks wrong.

Reply:
So, was the original name correct in this case?

Reply:
No, it is weird too. Hopefully the Scott topology typeclass will let you find a better name.

Reply:
> No, it is weird too. Hopefully the Scott topology typeclass will let you find a better name.

@YaelDillies  The typeclass? Am I to infer from this that you think this result should be stated as:
```
lemma nameTBD [TopologicalSpace α] [ScottTopology α] [TopologicalSpace β] [ScottTopology β]
  (f : α → β) :
  ScottContinuous f ↔ Continuous f := sorry
```
?

Reply:
Yeah, that sounds right.

Reply:
Or possibly?
```
lemma ContinuousEqScottContinuous [TopologicalSpace α] [ScottTopology α] [TopologicalSpace β] [ScottTopology β]
  (f : α → β) :
  Continuous f = ScottContinuous f :=
```

Reply:
No, your previous `scottContinuous_iff_continuous` looked best. And you should tag it simp.

Reply:
Presumably I should also change the results this result depends upon from `WithScottTopology` to `ScottTopology` e.g.
```
variable [TopologicalSpace β] [ScottTopology β]

lemma continuous_monotone {f : α → β}
  (hf : Continuous f) : Monotone f := sorry

lemma isOpen_iff_upper_and_LUB_mem_implies_inter_nonempty (u : Set α) :
IsOpen u ↔ (IsUpperSet u ∧ ∀ (d : Set α) (a : α), d.Nonempty → DirectedOn (· ≤ ·) d → IsLUB d a →
a ∈ u → (d ∩ u).Nonempty) := sorry
```

Reply:
Yep

Reply:
Okay, how's that?